reproc/src/reproc.c

```diff
@@ -651,6 +651,26 @@ int reproc_stop(reproc_t *process, reproc_stop_actions stop)
   return r;
 }
 
+int reproc_pid(reproc_t *process)
+{
+  ASSERT_EINVAL(process);
+  ASSERT_EINVAL(process->status != STATUS_IN_CHILD);
+  ASSERT_EINVAL(process->status != STATUS_NOT_STARTED);
+
+#ifdef _WIN32
+  // On windows the handle is a pointer to a HANDLE object.
+  // GetProcessId returns a DWORD which on win32 is a uint.
+  // However process IDs are never negative and its
+  // extremely unlikely a processID on windows would be
+  // larger than 200k.
+  return GetProcessId(process->handle);
+#else
+  // On posix return the handle directly as this is
+  // the process ID.
+  return process->handle;
+#endif
```

Comment:
This should be moved into a function in process.h (`process_pid`) that is implemented for both Windows and POSIX in process.windows.c and process.posix.c

Reply:
Sorry Dan didnt see the comment earlier. Ive made the changes and pushed another round. This builds and tests fine on the windows box I have access. 

thanks,

-mp